acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/PantheonNode.java

```diff
@@ -291,6 +300,10 @@ private NodeRequests nodeRequests() {
     return nodeRequests;
   }
 
+  public Web3jService web3jService() {
```

Comment:
Super not keen on breaking this encapsulation, is there really no other way to achieve this?

Reply:
done. no longer needed